### PR TITLE
eMIS removal from externalServices, testing responses, json data, and specs

### PR DESCRIPTION
**Pull request by rjohnson2011**
## Summary

This PR removes the following deprecated eMIS references from vets-website:
- eMIS as a dependency in form and external service configs
- eMIS as an external service in testing responses 
- eMIS in user json data
- Commented out eMIS code in server mocks
- Removal of an eMIS downtime cypress spec (this spec was attempted to pass with different external services in place, but each time the test resulted in a timeout so it was removed)

## Related issue(s)

- Zenhub ticket - https://app.zenhub.com/workspaces/platform-product-team-633af4074573d06c3cda142a/issues/gh/department-of-veterans-affairs/va.gov-team/75204
- Epic - https://app.zenhub.com/workspaces/platform-product-team-633af4074573d06c3cda142a/issues/gh/department-of-veterans-affairs/va.gov-team/75212

## Testing done

- All specs are passing after the following spec was removed - `src/applications/personalization/profile/tests/e2e/account-security/downtime.cypress.spec.js`

## Requested Feedback

Would appreciate input from FE team regarding the failing spec that was removed mentioned above. There are several other `maintenance_windows` specs that test other external services, so there's an assumption that this spec removed is particular to eMIS although confirmation of this assumption would be appreciated.

### CODE REVIEW

**Comment by rjohnson2011** [comment]
@adamwhitlock1 This PR is part of an epic to fully deprecate and remove eMIS from vets-api and vets-website. I removed one spec that was referencing eMIS and would not pass when trying other external services like 'evss' and 'arcgis' which also failed. Can you confirm that removing the spec in this PR (now that eMIS is removed) is acceptable and will not lead into any unanticipated breaking changes. Thanks!

**Review by Andrew565 — DISMISSED** [review]

File: src/applications/personalization/profile/tests/e2e/account-security/downtime.cypress.spec.js [hunk 1]
```diff
@@ -48,31 +45,4 @@ context('downtime notification cases for Account Security', () => {
...
-    cy.injectAxeThenAxeCheck();
-
-    cy.findByText(
-      'We can’t show your account security information right now.',
-    ).should('exist');
-
-    cy.findByText('Sign in information').should('not.exist');
-    cy.findByText('Account setup').should('not.exist');
-  });
 });
```
Comment (adamwhitlock1) [inline src/applications/personalization/profile/tests/e2e/account-security/downtime.cypress.spec.js:48]: I think we can still test this, if we remove the emis dependency that is passed to the downtime component within the page itself and test with mvi as the externalService.

This should do the trick.

```suggestion
  it('should show downtime active banner, and not show the page content', () => {
    cy.intercept('GET', '/v0/maintenance_windows', {
      data: [
        {
          id: '139',
          type: 'maintenance_windows',
          attributes: {
            externalService: 'mvi',
            description: 'My description',
            startTime: beforeNow,
            endTime,
          },
        },
      ],
    });
    cy.visit(PROFILE_PATHS.ACCOUNT_SECURITY);

    cy.injectAxeThenAxeCheck();

    cy.findByText(
      'We can’t show your account security information right now.',
    ).should('exist');

    cy.findByText('Sign in information').should('not.exist');
    cy.findByText('Account setup').should('not.exist');
  });
});
```

Comment (adamwhitlock1) [reply src/applications/personalization/profile/tests/e2e/account-security/downtime.cypress.spec.js:48]: Then if we update the [Account Security component](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/personalization/profile/components/account-security/AccountSecurity.jsx#L29)
and remove `emis` from the dependencies prop of the DowntimeNotification component that will clean it up the rest of the way for this page.

**Review by adamwhitlock1 — CHANGES_REQUESTED** [review]

File: src/applications/personalization/profile/tests/e2e/account-security/downtime.cypress.spec.js (hunk 1 above)
Comment (rjohnson2011) [reply src/applications/personalization/profile/tests/e2e/account-security/downtime.cypress.spec.js:48]: Thank your for your input @adamwhitlock1 ! Following the above suggestions!

File: src/applications/mhv/medical-records/tests/e2e/fixtures/non_mr_user.json [hunk 2]
```diff
@@ -127,14 +127,6 @@
...
-        "errors": [
-            {
-                "externalService": "EMIS",
-                "startTime": "2023-06-22T19:17:15Z",
-                "endTime": null,
-                "description": "EMISRedis::VeteranStatus::RecordNotFound, NOT_FOUND",
-                "status": 404
-            }
-        ]
+        "errors": []
```
Comment (rmtolmach) [inline src/applications/mhv/medical-records/tests/e2e/fixtures/non_mr_user.json:130]: Did you check the response in the UI to make sure it looks like this? I thought it was `"errors": null`

Comment (rmtolmach) [reply src/applications/mhv/medical-records/tests/e2e/fixtures/non_mr_user.json:130]: User 16 from [staging users](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv) doesn't have any errors and this is what I see:
image

I'm not sure how critical it is to get this response perfect, but I wouldn't want frontend code to be written to expect an empty `[]` when it's really `null`

File: src/applications/personalization/profile/tests/e2e/account-security/downtime.cypress.spec.js (hunk 1 above)
Comment (rjohnson2011) [reply src/applications/personalization/profile/tests/e2e/account-security/downtime.cypress.spec.js]: Note: I will be removing the dependencies prop of DowntimeNotification in [this ticket](https://app.zenhub.com/workspaces/platform-product-team-633af4074573d06c3cda142a/issues/gh/department-of-veterans-affairs/va.gov-team/75205) next.

**Review by adamwhitlock1 — APPROVED** [review]

**Comment by rjohnson2011** [comment]
@department-of-veterans-affairs/vfs-mhv-medical-records @department-of-veterans-affairs/my-education-benefits Can I get an approval from one of your team members so I can merge this please?

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**